### PR TITLE
Invincible Time（want to be merged）

### PR DESCRIPTION
**Pull request by RYQ-22**
为坦克出生时设置了3s的无敌时间
如图：
Invincible Time

### CODE REVIEW

**Comment by RYQ-22** [comment]
无敌时间内血条为蓝色，支持所有unit

File: CMakeSettings.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
+      "installRoot": "${projectDir}\\out\\install\\${name}",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": ""
+    }
+  ]
+}
```
Comment (LazyJazz) [inline CMakeSettings.json:15]: Delete this file, add it to .gitignore

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@
 在选择完成后，点击 `Create pull request` 按钮提交你的 PR。
 
 这一子任务不需要你提交有意义的代码，你只需要正确地创建分叉，随意提交一些改动，然后正确提交一个 PR 即可，然后请进入下一个子任务。
-
+（改动）
```
Comment (LazyJazz) [inline README.md:34]: delete this

File: src/battle_game/core/CMakeLists.txt [hunk 3]
```diff
@@ -3,7 +3,7 @@ list(APPEND LIBRARY_LIST ${lib_name})
 
 add_library(${lib_name})
 file(GLOB_RECURSE source_files *.cpp *.h)
-target_sources(${lib_name} PRIVATE ${source_files})
+target_sources(${lib_name} PRIVATE ${source_files}  )
```
Comment (LazyJazz) [inline src/battle_game/core/CMakeLists.txt:6]: why are u making this change?

File: src/battle_game/core/unit.h [hunk 4]
```diff
@@ -94,7 +95,8 @@ class Unit : public Object {
   glm::vec4 front_lifebar_color_{};
   glm::vec4 background_lifebar_color_{};
   glm::vec4 fadeout_lifebar_color_{};
-
+  float invincibleTime{3 * 60};
```
Comment (LazyJazz) [inline src/battle_game/core/unit.h:98]: lower_case_ ending with an underscore

**Review by LazyJazz — CHANGES_REQUESTED** [review]

File: src/battle_game/core/CMakeLists.txt (hunk 3 above)
Comment (RYQ-22) [reply src/battle_game/core/CMakeLists.txt:6]: 加文件时vs自动改的，没删干净（）

File: CMakeSettings.json (hunk 1 above)
Comment (RYQ-22) [reply CMakeSettings.json]: 已修改

**Review by LazyJazz — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 5]
```diff
@@ -6,4 +6,19 @@ build
...
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
+      "installRoot": "${projectDir}\\out\\install\\${name}",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": ""
+    }
+  ]
+}
```
[the remark this answers is not in the document]
Comment (RYQ-22) [reply .gitignore]: Issues are fixed.

**Review by qinchuanhui — APPROVED** [review]
You can get the score, but seems all of your PRs have some format ERROR (e.g.  in `gitignore` , `CMakeLists.txt`🤦‍♀️.

**Pull request closed without merging**